pygmt/datasets/load_remote_dataset.py

```diff
@@ -84,11 +80,10 @@ class GMTRemoteDataset(NamedTuple):
             "01m": Resolution("01m", registrations=["gridline"], tiled=True),
         },
     ),
-    "earth_free_air_anomaly": GMTRemoteDataset(
-        title="free air anomaly",
-        name="free_air_anomaly",
-        long_name="IGPP Earth Free-Air Anomaly",
-        units="mGal",
+    "earth_mag": GMTRemoteDataset(
+        name="earth_mag",
+        long_name="Earth Magnetic Anomaly Model (EMAG2)",
```

Comment:
https://www.generic-mapping-tools.org/remote-datasets/earth-mag.html
```suggestion
        long_name="EMAG2 Earth magnetic anomaly model",
```